content/3_Silver/Binary_Search_Ans.mdx

```diff
@@ -708,7 +723,7 @@ int main() {
 <JavaSection>
 
 ```java
-public static void main(String[] args) throws IOException
+public static void main(String['Binary Search'] args) throws IOException
```

Comment:
here

Reply:
I'm putting comments everywhere so if there's any that aren't "outdated" it's easy to tell if they aren't fisxed